api/src/main/java/com/velocitypowered/api/util/FastUuidSansHyphens.java

```diff
@@ -1,25 +1,8 @@
 /*
- * The MIT License (MIT)
+ * Copyright (C) 2018 Velocity Contributors
  *
- * Copyright (c) 2018 Jon Chambers
```

Comment:
I believe this was mistakenly removed by the license header plugin.

Reply:
We can still incorporate MIT code into a GPLv3 project. I will fix the copyright here later.